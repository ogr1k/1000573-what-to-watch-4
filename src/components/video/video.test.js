import React from "react";
import renderer from "react-test-renderer";
import Video from "./video.jsx";

const mock = {
  video: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`Render Video Film Card`, () => {

  const tree = renderer.create(<Video videoSrc={mock.video} onmouseout={() => {}}/>, {
    createNodeMock: () => {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
