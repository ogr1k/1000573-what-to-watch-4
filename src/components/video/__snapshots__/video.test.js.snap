// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Video Film Card 1`] = `<video />`;
